packages/rule-babel-config/tests/fixtures/no-babel-package-json/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
```

Comment:
Remove all the fields that are not required for this fixture so people don't misinterpret them.